schemas/container.json

```diff
@@ -206,6 +212,9 @@
                 "pattern": "^((?=[a-z0-9-_]{1,63}\\.)(xn--)?[a-z0-9_]+(-[a-z0-9_]+)*\\.)+[a-z]{2,63}$"
               },
               "uniqueItems": true
+            },
+            "ruleComponentSequencing": {
```

Comment:
I think "rule component" here might be too narrow a definition of what this will encompass. For example, we plan on eventually changing data elements so they can be asynchronous. When we do so, calls to `_satellite.getVar(...)` will return a promise for asynchronous data elements. Also, asynchronous data elements will wait on asynchronous data element dependencies. These things are not rule-component specific.

That said, I don't know if I have better suggestions. Maybe `sequencingEnabled` or `componentSequencingEnabled`? Do you know what we're planning on calling it in the UI?

Reply:
I renamed it to `sequencingEnabled`. I am not sure about the UI name yet.

Reply:
It might be best to discuss with Ben what it's going to be called in the UI before we settle on API names. Not that they must be the same, but it would probably be nice if they were.

Reply:
For me `sequencingEnabled` seems to not be very specific. I don't think that async data elements will be executed in sequence.

Reply:
Okay, so @dompuiu and I just had a talk with Ben. We're going to go with `ruleComponentSequencing`. Sorry to make you switch it back.

Reply:
@dompuiu can you take this PR from here?

Reply:
Can we make `ruleComponentSequencing` required?

Reply:
If we make it required, this would need to be released with the `next` tag, if we are still doing this. If not, the release need to be synchronized with the Launch services.

Reply:
Let's use the `next` tag.